conf/layer.conf

```diff
@@ -23,4 +23,4 @@ BBFILES += "${@' '.join('${LAYERDIR}/layers/%s/recipes*/*/*.bbappend' % layer \
 BBFILES += "${@' '.join('${LAYERDIR}/layers/%s/recipes*/*/*.bb' % layer \
```

Comment:
layer.conf: add Warrior to LAYERSERIES_COMPAT

Reply:
fixe